sputnikdao2/tests/utils/mod.rs

```diff
@@ -133,6 +133,6 @@ pub fn vote(users: Vec<&UserAccount>, dao: &Contract, proposal_id: u64) {
     }
 }
 
-pub fn to_va(a: AccountId) -> ValidAccountId {
-    ValidAccountId::try_from(a).unwrap()
+pub fn to_va(a: AccountId) -> AccountId {
+    AccountId::try_from(a).unwrap()
 }
```

Comment:
optionally, can just remove this altogether, this is essentially a no-op. (pain to remove out so I get why not)

Reply:
Removed!